src/components/pages/home/index.tsx

```diff
@@ -44,24 +52,95 @@ function HomePage() {
 
   return (
     <>
-      <nav className="py-4 shadow-md dark:shadow-gray-600">
-        <div className=" max-w-screen-xl mx-auto flex flex-row justify-between items-center">
-          <Link href="/">
+    
+    <nav className="py-4 shadow-md dark:shadow-gray-600">
+      <div className="max-w-screen-lg mx-auto flex flex-row justify-between items-center">
+        <Link href="/">
             <Image
               src="/assets/logo.png"
               alt="profile logo"
               width={50}
               height={50}
               className="dark:shadow-md dark:shadow-gray-500 rounded-full"
             />
+        </Link>
+
+        <div className="md:hidden">
+        <Theme />
+        </div>
+        
+
+        {/* Hamburger menu button for small screens */}
+        <div className="md:hidden flex items-center">
+          <button
+          onClick={handleMenuClick}
+          className={`lg:hidden dark:text-white dark:hover:text-primary focus:outline-none ${isMenuOpen ? 'hidden' : 'closed'} text-gray-600 hover:text-primary`}
+          >
+              <svg
```

Comment:
instead of svg, can you use react-feather as it is installed already and everything will be in sync